website/content/docs/agent/template.mdx

```diff
@@ -194,36 +211,18 @@ using the certificates `validTo` field.
 This does not apply to certificates generated with `generate_lease: true`. If set
 Vault Agent template will apply the non-renewable, leased secret rules.
 
-## Auto Auth and Templating Example
-
-The following demonstrates configuring Vault Agent to template secrets using the
-AppRole Auth method:
-
-```python
-pid_file = "./pidfile"
-
-vault {
-  address = "https://127.0.0.1:8200"
-}
-
-auto_auth {
-  method {
-    type = "approle"
+When Agent's auto-auth triggers a re-authentication, due to a token expiry for
```

Comment:
```suggestion
When Agent's auto-auth re-authenticates, due to a token expiry for
```

Reply:
Maybe also worth explicitly saying what people would need to do to avoid this churn? i.e. give the auth method used by auto-auth a longer token TTL

Reply:
This is dependent on whether the token is a periodic or batch token as well (which disallows renewals entirely). Some users may not want their tokens to be long-lived. I don't think there's a way for us to consistently make this recommendation.